src/components/AppLayout/AppLayout.tsx

```diff
@@ -12,6 +12,7 @@ import { useStyles } from "./styles";
 interface AppLayoutProps {
   children: React.ReactNode;
   fullSize?: boolean;
+  disableBorder?: boolean;
```

Comment:
I think it's better to swap it

```
border?: boolean
```

It's better to pass `border: false` than `disableBorder: true` imo

Reply:
turns out this was leftover and it is not needed now